### ISSUE TITLE
Peer list stale?

### ISSUE THREAD

**Issue by theCheek** [cardigann/cardigann#57]
Hi, there seems to be a discrepancy between the seeders/peers that Sonarr says are available for a torrent and what is actually available. Please see the screenshots attached. 

This is resulting in Sonarr selecting dead torrents, or when manually selecting torrents from within Sonarr wrong choices are being made.

Thanks

![cardigann1](https://cloud.githubusercontent.com/assets/3003042/19028204/30cdf490-8942-11e6-9bfa-450353abd305.PNG)
![cardigann2](https://cloud.githubusercontent.com/assets/3003042/19028205/30cf207c-8942-11e6-9725-d708307ac768.PNG)

**Comment by mmarquezs** [comment]
Which tracker? Now cardigann gets the seeders/peers from parsing the webpage, so this could be an error parsing the page (getting the wrong data from the page) or the page of the trackers shows wrong data or could be an other thing.

**Comment by theCheek** [comment]
It was pulling the feed from the aggregate, which pulled it from IPT. Perhaps the aggregate doesnt update fast enough? I figured I stead of putting each individual tracker (via cardigann)  into sonarr, I could put the aggregate feed to speed up searching. Is this wrong?

**Comment by lxt2** [comment]
There is no caching involved at any step.

**Comment by lxt2** [comment]
I'll see if I can replicate this.

**Comment by lxt2** [comment]
So to be clear, the discrepancy is +/- 1 peer? Could not that just have happened during the time when the screenshots were taken?

**Comment by theCheek** [comment]
@lxt2 no, the discrepancy is +/- 28 peers. In my OP there are three screenshots - the first being the one taken after a manual search in sonarr (peers 28/0), the second from the torrent website's page (0/1), and the third from Rutorrent (0/0).

**Comment by lxt2** [comment]
Ah, right, sorry I missed the 28 peers in the first one. I'll see if I can replicate on IPT.

**Comment by lxt2** [comment]
I'm still struggling to replicate this, any chance you could post what you were searching for?

**Comment by Githubtordl** [comment]
I think this is the same or similar problem I'm having, Cardigann isn't showing any seeders/leechers on Sonarr, it's always on 0/0, tested on PB & EzTV trackers. I've compared the same PB torrents on Jackett, it's showing the number of seeders/leechers fine. Is this a bug?

**Comment by lxt2** [comment]
No, it's not the same problem @Githubtordl. See #79, which is solved in 1.8.2. Please update and check if the problem goes away.

**Comment by Githubtordl** [comment]
Oops! I can confirm that my issue is solved in 1.8.5, thanks

**Comment by theCheek** [comment]
@lxt2 sorry for the delay, I was away on business last week.
I can reproduce the error pretty much every time I search for anything using Manual Search in Sonarr. 

Just did so with the below torrents.
![ipt seeding disparity github 57 -sonarr](https://cloud.githubusercontent.com/assets/3003042/19227102/979e6378-8ebd-11e6-8b3f-1f6465a7746a.PNG)
The above is when manually searching within Sonarr. Notice seeders is showing as 32/0

![ipt seeding disparity github 57 -ipt](https://cloud.githubusercontent.com/assets/3003042/19227105/a4d4890a-8ebd-11e6-8067-d8dd0ed67652.PNG)
The above is when clicking on the link provided in the results screen from Sonarr. Notice seeders is 0/0.

**Comment by theCheek** [comment]
This problem seems to still exist for IPTorrents. Please see screenshots below which a resulting in stale torrents being sent to Rutorrent

![iptorrents cardigann issue 1](https://cloud.githubusercontent.com/assets/3003042/21960614/ec8bd8a6-daf9-11e6-8c4b-45ac805fd3d2.PNG)

![iptorrents cardigann issue 2](https://cloud.githubusercontent.com/assets/3003042/21960617/fa06ce8c-daf9-11e6-99b1-3ce57fdef784.PNG)
